### DISCUSSION TITLE
Being able to see who added you as a friend

### DISCUSSION THREAD

**Discussion by bfuhriman** [ppy/osu#27022] [Ideas] [answered]
There are some old feature requests for this that gained a lot of attention ([here](https://osu.ppy.sh/community/forums/topics/130231?n=1) and [here](https://osu.ppy.sh/community/forums/topics/65468?n=1)). I found some old replies from peppy on the subject where he said he doesn't want osu! to turn into another social media platform and this feature was missing by design. I understand that viewpoint but those replies are 7+ years old now so I thought there's a chance it could be reconsidered.

This could be implemented several ways, but I think it would be most efficient with a notification when someone adds you and an additional tab on [this](https://osu.ppy.sh/home/friends) page to see people who have added you/sent a friend request. For popular players who would be bombarded with requests, the notifications could be optional with a function allowing you to search through requests, or there could be a way to make your profile private/not accepting requests.

**Comment by Rekunan** [comment]
For now, there's this, which only works if the other person is also registered on it, but luckily it's pretty popular: https://mutualify.stanr.info/

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/5227
